pages/curated-sets/[id].js

```diff
@@ -106,6 +108,20 @@ export default function CuratedSet({
                   </DataItemValue>
                 </>
               )}
+              {curatedSet.assemblies?.length > 0 && (
+                <>
+                  <DataItemLabel>Assemblies</DataItemLabel>
+                  <DataItemValue>{curatedSet.assemblies}</DataItemValue>
+                </>
+              )}
+              {curatedSet.transcriptome_annotations?.length > 0 && (
+                <>
+                  <DataItemLabel>Transcriptome Annotations</DataItemLabel>
+                  <DataItemValue>
+                    {curatedSet.transcriptome_annotations}
```

Comment:
For `curatedSet.assemblies` and `curatedSet.transcriptome_annotations`, need to convert the array to string first before put it in DataItemValue, for example: `{curatedSet. assemblies.join(", ")}`

Reply:
fixed. thanks I forgot.